web/EXAMPLES.md

```diff
@@ -0,0 +1,166 @@
+
+We have `default: block`, so we block all third party requests that don't match one of the rules. The first rule doesn't have any `exceptions` so we block all matching requests. It also has a surrogate[^2] listed which, if supported by the client, can be substituted in place of the request.
+The second rule is limited to blocking all matching requests except for those whose type is `image`.
+
+[^2]: Certain tracking scripts are implemented in a way that attaches function calls to page elements. When these scripts are blocked, they can cause usability issues. In order to block these requests while still maintaining site functionality, we redirect the requests to surrogate code that replaces all of their functions with no-ops.
+
+```json
+{
+    "domain": "test-tracker.net",
+    "rules": [
+        {
+            "rule": "test-tracker\\.net\\/instream\\/.*\\/ad_status\\.js",
+            "surrogate": "ad_status.js"
+        },
+        {
+            "rule": "test-tracker\\.net\\/ddm\\/",
+            "exceptions": {
+                "types": [
+                    "image"
+                ]
+            }
+        }
+    ],
+    "default": "block",
+    "owner": {
+        "name": "Tracking Company"
+    }
+ }
+```
+
+|  Site | Request URL  | Request type | Block | Reason |
+|---|---|---|---|---|
+| example.com | test-tracker.net/instream/1234/ad_status.js |  script |  true | matches rule |
+| example.com | test-tracker.net/ddm/ | script  |  true | matches rule |
+| example.com | test-tracker.net/ddm/ | image  |  false | matches rule and matches rule exception |
+| example.com | test-tracker.net/adimage.png | image  |  true | no matching rule, default is set to 'block' |
+
+### Rule with `exceptions`
+
+We have `default: ignore`, so we only block requests that match a rule. The first rule doesn't have any `exceptions`, so we block any matching request. The second rule has a blocking exception. We will not block matching `image` requests that come from `test-site-2.com` or its subdomains.
+
+```json
+{
+    "domain": "example.net",
+    "rules": [
+        {
+            "rule": "connect\\.example\\.net\\/signals\\/"
+        },
+        {
+            "rule": "example\\.net\\/.*\\/AudienceNetworkVPAID\\.",
+	    "exceptions": {
+	    	"types": [
+		    "image"
+		], 
+		"domains": [ 
+		    "test-site-2.com"
+		]
+	    }
+        }
+    ],
+    "default": "ignore",
+    "owner": {
+        "name": "Example Tracker"
+    }
+ }
+```
+
+|  Site | Request URL  | Request type | Block | Reason |
+|---|---|---|---|---|
+| test-site.com | connect.example.net/signals/ |  script |  true | matches rule |
+| test-site.com | example.net/tracker.js |  script |  false | default set to 'ignore' |
+| test-site-2.com | example.net/123/AudienceNetworkVPAID.png | image  |  false | matches exception type and domain |
+| test-site-2.com | example.net/123/AudienceNetworkVPAID.png | script  |  true | matches exception domain, but not type |
```

Comment:
is it deliberately `AudienceNetworkVPAID.png` again ? I guess it's possible but probably adds confusion